src/main/java/de/recipe/controller/RecipeController.java

```diff
@@ -0,0 +1,73 @@
+package de.recipe.controller;
+
+import de.recipe.model.Recipe;
+import de.recipe.service.RecipeServiceImpl;
+import de.recipe.web.RecipeWeb;
+import de.recipe.web.RecipeWebOutput;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class RecipeController {
+
+    private RecipeServiceImpl recipeService;
+
+    @Autowired
+    public RecipeController(RecipeServiceImpl recipeService) {
+        this.recipeService = recipeService;
+    }
+
+    @GetMapping( value = "/recipes" )
+    public List <RecipeWebOutput> getAllRecipe() {
+        return recipeService.getAllRecipe();
+    }
+
+    @PostMapping( value = "/recipes", consumes = "application/json" )
```

Comment:
No need for `consumes = "application/json" `, should work without it
```suggestion
    @PostMapping( value = "/recipes")
```